src/ivoc/ivocmain.cpp

```diff
@@ -249,7 +249,8 @@ void mac_open_app(){
 
 extern "C" {
 	int ivocmain(int, const char**, const char**);
-	int (*p_neosim_main)(int, const char**, const char**);
+        int ivocmain_session(int, const char**, const char**, int start_session);
```

Comment:
```suggestion
      int ivocmain_session(int, const char**, const char**, int start_session);
```
spacing

Reply:
Actually, I don't know why p_neosim_main was removed.

Reply:
It wasn't - just moved one line down :) github's diff is playing games with us!